examples/AllTests/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+enable_testing()
```

Comment:
This should typically be called in the projects root path. According to the [cmake docs](https://cmake.org/cmake/help/latest/command/enable_testing.html) it's not necessary if the CTest package is included (it's in the root CMakeLists.txt).

Reply:
Interestingly changing this for the tests breaks ctest :confused:

Reply:
I have opened a PR #1351 to fix this.